source/fundamentals/indexes.txt

```diff
@@ -397,6 +397,34 @@ Refer to the :manual:`Unique Indexes page </core/index-unique>` in the MongoDB s
 
 .. driver-content-end
 
+Clustered Indexes
+~~~~~~~~~~~~~~~~~
+
+**Clustered indexes** are indexes that improve the performance of
+insert, update, and delete operations on **clustered collections**.
+Clustered collections store documents ordered by the clustered index key
+value.
```

Comment:
The part _"clustered indexes are indexes that improve the performance of ..."_ appears to be posed as the essence of what it means for an index to be clustered. However, it is actually not, and the essence is that the presence of a clustered index makes the collection to store documents ordered by the key (such a collection is called a clustered collection). I propose to specify only the essence on this page and refer users to the server documentation for any other details.